index.html

```diff
@@ -12,10 +12,101 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    
+    <nav class ="navbar">
+        <div class="content" >
+            
+            <ul class="link">
+                <li class="active">Meet Karma</li>
+                <li><a href="How it works">How it works</a></li>   
+                <li><a href="Store">Store</a></li>   
+                <li><a href="Blog">Blog</a></li>   
+                <li><a href="Help">Help</a></li>
+                <li><a href="Login">Login</a></li>        
+          
+            </ul>
+              <img class="header-image" src="/img/karma-logo.svg" alt="image logo" width="26" /> 
```

Comment:
The image will currently not display as you are using an absolute link to the svg file. i.e you are putting a '/' in front of the link declaration. If you look in the dev tools you will see it is looking for the image from the following uri: file:///img/karma-logo.svg which doesnt exist. A very easy fix you just need to remove the '/'. so it would be

<img class="header-image" src="img/karma-logo.svg" alt="image logo" width="26" />